### PR TITLE
Abel - Add default namespace

### PR DESCRIPTION
**Pull request by unoctium1**
New scripts default to game jam cat namespace, also bump packages to latest versions

### CODE REVIEW

File: Packages/manifest.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "com.unity.ide.rider": "2.0.7",
     "com.unity.ide.visualstudio": "2.0.5",
     "com.unity.ide.vscode": "1.2.3",
-    "com.unity.test-framework": "1.1.20",
-    "com.unity.textmeshpro": "3.0.1",
-    "com.unity.timeline": "1.2.17",
+    "com.unity.test-framework": "1.1.22",
```
Comment (m4rtint) [inline Packages/manifest.json]: we can probably remove the timeline and test framework, since we won't be using those anyway. but nbd. it's up to you.

**Review by m4rtint — DISMISSED** [review]
lgtm

File: Packages/manifest.json (hunk 1 above)
Comment (unoctium1) [reply Packages/manifest.json]: I could see timeline coming in handy cause we're doing stuff with cinemachine and potentially will want to throw some audio in alongside the cinemachine transitions, but yeah updating the packages was more just habit for starting new projects haha, I'd be fine with getting rid of the test framework or collaborate

Comment (m4rtint) [reply Packages/manifest.json]: yeah test and collaborate can be removed

**Review by Gursaanj — APPROVED** [review]
Looks Good

**Review by natibeltran777 — APPROVED** [review]

**Pull request merged**